compiler/rustc_metadata/src/creader.rs

```diff
@@ -239,47 +239,49 @@ impl CStore {
             );
         }
     }
+
+    pub fn new(sess: &Session) -> Self {
+        let mut stable_crate_ids = FxHashMap::default();
+        stable_crate_ids.insert(sess.local_stable_crate_id(), LOCAL_CRATE);
+        Self {
```

Comment:
```suggestion
        CStore {
```
Nit: could you avoid using `Self` in non-generic code, it makes searching and reading code harder.

Reply:
I guess I have rust-analyzer brain by now ^^ I have the exact opposite opinion on usage of Self (namely use it wherever possible)

Reply:
I addressed this